tests/lib/install_test_helpers.sh

```
@@ -0,0 +1,163 @@
+  if [ -x "$path" ]; then
+    pass "$desc"
+  else
+    fail "$desc (実行権限なし: $path)"
+  fi
+}
+
+# --- セットアップ / クリーンアップ ---
+
+create_test_repo() {
+  TMPDIR_ROOT=$(mktemp -d)
+  cd "$TMPDIR_ROOT"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)